src/re_frame/subs.cljs

```diff
@@ -79,7 +79,7 @@
      (let [query-id   (first-in-vector v)
            handler-fn (get @qid->fn query-id)]
        (when ^boolean js/goog.DEBUG
-         (when-let [not-reactive (remove #(implements? reagent.ratom/IReactiveAtom %) dynv)]
```

Comment:
I did wonder about this when I read the code, but assumed that when-let did nil punning.